bin/run_docker_test

```diff
@@ -71,6 +75,11 @@ def main():
         try:
             print("Running: {}".format(compose_up))
             run(compose_up, check=True, timeout=timeout)
+            handle = Popen(args=compose_logs, stdout=PIPE,
```

Comment:
I think you may also want to add `, stderr=PIPE` and add `sys.stdout.write(handle.stderr.read())` below.